### DISCUSSION TITLE
How to send SMS notifications to multiple numbers.

### DISCUSSION THREAD

**Discussion by devendra-rgb** [upptime/upptime#942] [Q&A]
I can only send messages to only phone number how to trigger it to multiple numbers.

**Comment by titanism** [comment]
I suppose you could send to a number that then forwards to multiple others

**Comment by titanism** [comment]
PR submitted at https://github.com/upptime/uptime-monitor/pull/259

**Reply by Delta-Trace** [reply]
Hello titanism,

This seems to be an issue with all notification providers I posted a discussion two days ago #1045 about only being able to send notification to one email.

May you broaden your PR to comma separate all recipient lists?

**Reply by titanism** [reply]
You can submit a PR if you want that @Delta-Trace.  We have no need for others, so we're not going to.
